### PR TITLE
CODE RUB: Action variable updated fix in testing exceptions for retrieve all calendar entries

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

**Comment by hassanhabib** [comment]
@deepshri7 please fix the conflicts

**Comment by hassanhabib** [comment]
@deepshri7 please fix conflict issue

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -24,14 +24,14 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntriesWhenSqlExc
             this.storageBrokerMock.Setup(broker =>
                 broker.SelectAllCalendarEntries())
                     .Throws(sqlException);
-            
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs:27]: Run code cleanup

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -61,12 +61,12 @@ public void ShouldThrowServiceExceptionOnRetrieveAllCalendarEntriesWhenException
                     .Throws(serviceException);
 
             // when
-            Action retrieveAllCalendarEntryAction = () =>
-                this.calendarEntryService.RetrieveAllCalendarEntries();
+            Action retrieveAllCalendarEntriesAction = () =>
+            this.calendarEntryService.RetrieveAllCalendarEntries();
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs:65]: Fix the tab here and below.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs (hunk 2 above)
Comment (ShriHumrudha) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntries/CalendarEntryServiceTests.Exceptions.RetrieveAll.cs:65]: @deepshri7 this needs attention

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**